bin/update-dep-versions.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+// update all the dependencies inside packages/node_modules/*/package.json to reflect
+// the version numbers in the top-level package.json
+
+var fs = require('fs');
+
+var topPkg = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+
+var modules = fs.readdirSync('./packages/node_modules');
+modules.forEach(function (mod) {
+  var pkgPath = './packages/node_modules/' + mod + '/package.json';
+  var pkg = JSON.parse(fs.readFileSync(pkgPath, 'utf8'));
+  var depLists = [
+    pkg.dependencies, 
+    pkg.peerDependencies, 
+    pkg.optionalDependencies
+  ];
```

Comment:
That's a TODO. :) We could seriously just use a template for the LICENSE, README, and package.json.